extensions/notebook/package.json

```diff
@@ -329,6 +329,15 @@
           "Notebook"
         ],
         "configuration": "./language-configuration.json"
+      }, {
+        "id": "dib",
+        "extensions": [
+          ".dib"
+        ],
+        "aliases": [
+          ".NET Interactive Notebook"
+        ],
+        "configuration": "./language-configuration.json"
```

Comment:
Is this the language config for JSON?

Reply:
Yes, it's the same one we use for the notebook format. However, we never actually see a raw dib editor, so as far as I know this won't be used. Regardless, it'll be getting replaced eventually once we add this language contrib to the interactive extension.